backend/controllers/iepHistory.js

```diff
@@ -0,0 +1,56 @@
+const moment = require('moment')
+
+const getIepHistoryFactory = elite2Api => {
+  const getIepHistory = async (context, offenderNo) => {
+    const bookingDetails = await elite2Api.getDetails(context, offenderNo)
+    const iepSummary = await elite2Api.getIepSummaryWithDetails(context, bookingDetails.bookingId)
+
+    // Offenders are likely to have multiple IEPs at the same agency.
+    // By getting a unique list of users and agencies, we reduce the duplicate
+    // calls to the database.
+    const uniqueUserIds = [...new Set(iepSummary.iepDetails.map(details => details.userId))]
```

Comment:
think you might be able to drop the first summary detail by doing something like this up here:
`const [summaryDetail, ...iepDetails] =  iepSummary.iepDetails;`

and then refer to iepDetails in all the mapping below... might be a bit more of an explicit way of ignoring the first record, and means you don't need to mutate state with the shift below.